packages/lib/shim-init-node.ts

```diff
@@ -304,12 +304,10 @@ function shimInit(options: ShimInitOptions = null) {
 			...options,
 		};
 
-		const readChunk = require('read-chunk');
-		const imageType = require('image-type');
-
 		const isUpdate = !!options.destinationResourceId;
 
 		const uuid = require('./uuid').default;
+		const { fromFile } = await import('file-type');
```

Comment:
And should be `fromFile as fileTypeFromFile`

Reply:
[Dynamic imports returns a promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#return_value) but since the import should be at the top of the file, this doesn't matter here.

Reply:
> For new code all imports should be at the top of the file. Also can we not "await" the import? Not sure what it's for, but I don't see the need for it - imports are done synchronously.

I think the `await import` is to allow importing ESM from CommonJS. @pedr Does `await import` allow us to use an up-to-date version of the library?